### ISSUE TITLE
Multiselect with right click

### ISSUE THREAD

**Issue by eandmsz** [fman-community/fman#389]
Would be great enhancement to select files/directories with right click + drag, just like in Total Commander:
Operation > Mouse selection mode: Use right mouse button (as in NC) - context menu after 1 second

**Comment by JayBeeDe** [comment]
Yes it could be nice to be able to at least multiselect in order to open all the file somewhere

**Comment by mherrmann** [comment]
@JayBeeDe you can leave a thumbs-up :+1: on issue descriptions to have them moved up in priority. That's easier for me to manage than reading comments :-)

**Comment by m4rkus** [comment]
Is it possible to add this functionality by writing a plugin?

**Comment by mherrmann** [comment]
No, the API doesn't make it possible at the moment, I'm sorry.
